src/components/registerPage.js

```diff
@@ -1,17 +1,16 @@
-import React, { Component } from 'react';
-import { connect } from 'react-redux';
-import {bindActionCreators} from "redux";
-import {registerUserAction} from "../actions";
-import {Jumbotron} from "react-bootstrap";
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { bindActionCreators } from "redux";
+import { registerUserAction } from "../actions";
+import { Jumbotron } from "react-bootstrap";
 import Container from "react-bootstrap/Container";
 import Col from "react-bootstrap/Col";
 import Button from "react-bootstrap/Button";
 import Form from "react-bootstrap/Form";
-import {Link} from "react-router-dom";
-
+import { Link } from "react-router-dom";
 
 class RegisterPage extends Component {
```

Comment:
This component isn't using class properties, so it can be re-written as a functional component

Reply:
ok fine